test/e2e/suite-ingress/suite-ingress-features/resourcesync.go

```diff
@@ -20,10 +20,10 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/apache/apisix-ingress-controller/pkg/id"
 	ginkgo "github.com/onsi/ginkgo/v2"
 	"github.com/stretchr/testify/assert"
 
-	"github.com/apache/apisix-ingress-controller/pkg/id"
```

Comment:
We don't need to modify this line?

Reply:
test/e2e has its own separate go mod, so for it `apache/apisix-ingress-controller` is a 3rd party module